bedrock/firefox/templates/firefox/browsers/quantum.html

```diff
@@ -0,0 +1,55 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/browsers/quantum" %}
```

Comment:
There's no need to use `add_lang_files` for Fluent pages, so this can be removed.